SimPEG/Examples/MT_1D_ForwardAndInversion.py

```diff
@@ -1,9 +1,11 @@
 import SimPEG as simpeg
 import numpy as np
-import SimPEG.MT as MT
+from SimPEG import NSEM
```

Comment:
I would suggest that NSEM be imported from EM, 
```
from SimPEG.EM import NSEM 
```
(same as how FDEM, TDEM are currently treated)